netci.groovy

```diff
@@ -28,30 +25,35 @@ platformList.each { platform ->
     def buildCommand = '';
     def osForGHTrigger = os
     def version = "latest-or-auto"
+    def dockerRepository = "microsoft/dotnet-buildtools-prereqs"
+    def dockerContainer = ''
+    def dockerWorkingDirectory = "/src/core-setup"
+    def dockerCommand = ''
+    def portableArgs = ''
 
     // Calculate build command
     if (os == 'Windows_NT') {
         buildCommand = ".\\build.cmd -ConfigurationGroup=${configuration} -TargetArchitecture=${architecture}"
+        if ((architecture == 'arm' || architecture == 'arm64')) {
+            buildCommand += " -PortableBuild=true -Skiptests=true"
+        }
     }
-/*
-    else if ((os.startsWith("Ubuntu") || os.startsWith("Tizen")) &&
-             (architecture == 'arm' || architecture == 'armel')) {
-        def linuxcodename = '';
-        if (os == 'Ubuntu') {
-            version = "arm-cross-latest"
-            linuxcodename = 'trusty'
+    else if (os.startsWith("Ubuntu")) {
+        if ((architecture == 'arm' || architecture == 'armel')) {
+            if (os == 'Ubuntu') {
+                dockerContainer = "ubuntu-14.04-cross-0cd4667-20172211042239"
+            }
+            else if (os == 'Ubuntu16.04') {
+                dockerContainer = "ubuntu-16.04-cross-ef0ac75-20175511035548"
+            }
+            portableArgs = " -portable cross skiptests disablecrossgen"
         }
-        else if (os == 'Ubuntu16.04') {
-            version = "latest-or-auto-docker"
-            linuxcodename = 'xenial'
+        else {
+            dockerContainer = "ubuntu-14.04-debpkg-e5cf912-20175003025046"
+            buildCommand = "./build.sh --configuration ${configuration} --docker ubuntu.14.04"
```

Comment:
I was under the understanding  ` --docker ubuntu.14.04` should be removed, right?  this doesn't do anything anymore.